actionview/lib/action_view/dependency_tracker.rb

```diff
@@ -2,6 +2,9 @@
 
 require "concurrent/map"
 require "action_view/path_set"
+require "action_view/render_parser"
+require "action_view/ripper_tracker"
+require "action_view/erb_tracker"
```

Comment:
We can register an autoload instead in `action_view.rb`

Reply:
That is way better! thanks!